build.sh

```diff
@@ -99,10 +99,11 @@ done
 # Clean if requested
 #
 if [ -n "$CLEAN" ]; then
-	b_log "Cleaning build dir"
+	b_log "Cleaning build dirs"
 	rm -rf "$PREFIX_BUILD" "$PREFIX_BUILD_HOST"
 	rm -rf "$PREFIX_FS"
-	#FIXME: this should also remove unpacked sources in 'ports' (the easiest option is to move them to _boot)
+	rm -rf "$PREFIX_BOOT"
+	#FIXME: this should also remove unpacked sources in 'ports' (the easiest option is to move them to PREFIX_BOOT)
```

Comment:
remove fixme, sources are moved to _build

Reply:
Done.